app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/controllers");
+
+app.use(express.json());
```

Comment:
We dont have any end points that require a body yet, this should not be added until it needs to be used